kubernetes/helm_charts/networkconfig/values.j2

```diff
@@ -6,6 +6,7 @@ domain_name: {{ domain_name }}
 cloud_storage_url: {{ cloud_storage_url }}
 swarm_kibana_ip: {{groups['swarm-dashboard'][0]}}
 plugin_container_name: {{plugin_container_name}}
-offline_installer_container_name: {{ namespace }}-offlineinstaller
+offline_installer_container_name: {{ offline_installer_container_name }}
 merge_domain_status: {{ merge_domain_status | lower }}
 merge_proxy_server_name: {{ merge_proxy_server_name | d('""') }}
+offline_cloud_storage_url: {{ sunbird_offline_azure_storage_account | d('"sunbird"') }}.blob.core.windows.net
```

Comment:
Change the LHS to desktop_blob_url. I belive we are using the RHS in nginx, if so, dont change the RHS

Reply:
its a new variable all together.